less/_toolbar.less

```diff
@@ -125,23 +125,14 @@
 		}
 	}
 	.folder {
-		.txt-link {
+		&.not-empty {
 			.icon {
-				position: absolute;
-				left: 2px;
-
-				width: 27px;
-				height: 15px;
-
-				background-color: @usernav-item-folder-bgcolor;
-				background-image: url(@usernav-item-folder-icon);
-				background-position: 8px -1px;
-				background-repeat: no-repeat;
+				background-position: 0 -101px;
```

Comment:
Is this background-position just going to cause issues down the road? Seems like it can be removed, but then we almost have nothing in this declaration block, so maybe the whole thing just needs to be removed?